.github/workflows/vuln-scans.yaml

```diff
@@ -0,0 +1,41 @@
+name: Vulnerability scans
+on:
+  workflow_dispatch: {}   # Allow for manual triggers
+  schedule:
+    - cron:  '0 8 * * *'  # Daily, at 8:00 UTC
+  pull_request:
+
+jobs:
+  trivy-scan:
+    name: Trivy security scan
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy scan on repo
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
```

Comment:
[nit] I think this is the same
```suggestion
          scan-type: fs
```
etc. let's keep it consistent and drop the single quotes where we can, please.